examples/depthEstimation-single-image/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>ml5.js | Depth Estimation Single Image</title>
```

Comment:
Nitpick: the existing examples all have their title in such a format fwiw: `ml5.js faceMesh Bounding Box Example`